_includes/overall-schedule.md

```diff
@@ -25,3 +25,90 @@ Organizers will inform participants of the week schedule by email.
 {%- assign w-desc = "" %}
 {%- endfor %}
 {%- endfor %}
+
+<script type="application/ld+json" >
+{
+  "@context": "https://schema.org",
+  "@id": "https://4000-docelijah-openlifescien-imqv6d2ud7l.ws-eu71.gitpod.io/ols-6/schedule/",
```

Comment:
It should not be the URL to your GitPod but the global one

Reply:
Oops!